packages/core/src/tests/useTestDatabase.ts

```diff
@@ -1,20 +1,9 @@
-import {
-  close,
-  patchPgForTransactions,
-  rollbackTransaction,
-  startTransaction,
-  unpatchPgForTransactions,
-} from 'pg-transactional-tests'
+import { testTransaction } from 'pg-transactional-tests'
 import { afterAll, afterEach, beforeAll, beforeEach } from 'vitest'
 
 export default function useTestDatabase() {
-  beforeAll(async () => {
-    patchPgForTransactions()
-  })
-  beforeEach(startTransaction)
-  afterEach(rollbackTransaction)
-  afterAll(async () => {
-    unpatchPgForTransactions()
-    await close()
-  })
+  beforeAll(testTransaction.start)
+  beforeEach(testTransaction.start)
+  afterEach(testTransaction.rollback)
+  afterAll(testTransaction.close)
```

Comment:
I bumped the version of this package and they changed the way is setup. 
If doesn't work I'll move back to prev version

https://github.com/romeerez/pg-transactional-tests/commit/79c60a96b57676212d50b2dbc608ea83212fc63a